Apps/phsmime/readme.html

```diff
@@ -1,35 +1,46 @@
 <!-- File: readme.html
   Copyright (c) 2020 Splunk Inc.
 
-  SPLUNK CONFIDENTIAL - Use or disclosure of this material in whole or in part
-  without a valid written license from Splunk Inc. is PROHIBITED.
+  Licensed under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt)
 -->
 
 <html>
   <head></head>
   <body>
 
+    <h2>Licensing details for the app</h2>
+      <p><h3>M2Crypto</h3>
+        This app uses the M2Crypto module, which is licensed under the MIT License (MIT), Copyright (c) Ng Pheng Siong.
+      </p>
+      <p><h3>typing</h3>
+        This app uses the typing module, which is licensed under the Python Software Foundation License (PSF), Copyright (c) Guido van Rossum, Jukka Lehtosalo, Lukasz Langa, Ivan Levkivskyi.
+      </p>
+
     <h2><a id="Requirements_2"></a>Requirements</h2>
-    <p>Supported OS is CentOS 7+</p>
-    <p>Access your Phantom instance via Terminal and install following OS dependencies</p>
-    <pre><code class="language-bash">
-    <span class="hljs-comment"># Install library and its dependencies</span>
-    $~ sudo yum install python-devel openssl-devel gcc gcc-c++
-    </code></pre>
+      <p>Supported OS is CentOS 7+</p>
+      <p>Access your Phantom instance via Terminal and install following OS dependencies</p>
+      <pre>
+        <code class="language-bash">
+        <span class="hljs-comment"># Install library and its dependencies</span>
+        $~ sudo yum install python-devel openssl-devel gcc gcc-c++
+        </code>
+      </pre>
 
     <h2><a id="Keys_and_Certificates_3"></a>Keys and Certificates</h2>
-    <p>To use this app you need to provide an RSA key pair (this consists of a
-      public key and a private key) and an X.509 certificate of said public key.</p>
-    <p>To generate new RSA key pair, you could install and use OpenSSL on a CentOS 7
-      instance. Once properly installed, execute the following commands
-      to create key pairs and certificates.</p>
-    <pre><code class="language-bash">
-    <span class="hljs-comment"># Create a pair of keys</span>
-    $~ openssl req -newkey rsa:1024 -nodes -x509 -days 365 -out cert.pem
+      <p>To use this app you need to provide an RSA key pair (this consists of a
+        public key and a private key) and an X.509 certificate of said public key.</p>
+      <p>To generate new RSA key pair, you could install and use OpenSSL on a CentOS 7
+        instance. Once properly installed, execute the following commands
+        to create key pairs and certificates.</p>
+      <pre>
+        <code class="language-bash">
+        <span class="hljs-comment"># Create a pair of keys</span>
+        $~ openssl req -newkey rsa:1024 -nodes -x509 -days 365 -out cert.pem
```

Comment:
```suggestion
        $~ openssl req -newkey rsa:2048 -nodes -x509 -days 365 -out cert.pem
```

I'm not sure if this was added by the submitter, but I'm surprised to have the example key generation not use at least a 2048 bit RSA key.

Reply:
Yes, it's added by the submitter.

Reply:
Since I don't know if there is some underlying reason why they picked 1024, we can leave it as is. I would be curious to ask the submitter if they intentionally did so, but it's not critical.

Reply:
So, we are keeping the same for now. I have added the comment in the contributor's [MR](https://github.com/phantomcyber/phantom-apps/pull/227#discussion_r529793530) regarding it.